server/proto/heimdall.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package proto;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "/server/proto";
+
+service Heimdall {
+    rpc GetLatestSpan(GetLatestSpanRequest) returns (GetLatestSpanResponse) {}
+    rpc GetSpan(GetSpanRequest) returns (GetSpanResponse) {}
+    rpc GetEventRecords(GetEventRecordsRequest) returns (stream GetEventRecordsResponse) {}
+}
+
+message Validator {
+    uint64 ID = 1;
+    uint64 startEpoch = 2;
+    uint64 endEpoch = 3;
+    uint64 nonce = 4;
+    int64 power = 5;
+    string pubKey = 6;
+    string signer = 7;
+    string last_updated = 8;
```

Comment:
don't we have timestamp type to be used here?

Reply:
Okay, I used string because the `last_updated` we have here is in UNIX timestamp format
https://heimdall.api.matic.today/staking/validator/1

In the other `EventRecord` message of this proto I have used `google.protobuf.Timestamp` because it has normal timestamp format
https://heimdall.api.matic.today/clerk/event-record/36452

Reply:
Could we convert the string timestamp into time.Time or uint64 for Heimdall->Bor interaction and use google.protobuf.Timestamp ?

Reply:
Sure, Will check that and test.

Reply:
I checked and found that this is not even used in Bor
https://github.com/maticnetwork/bor/blob/88dbfa1c13c15464d3c1a3085a9f12d0ffb9b218/consensus/bor/heimdall/span/span.go#L8
Can we ignore the values which are not consumed by Bor then ?

Reply:
Removed